sda/cmd/rotatekey/rotatekey.md

```diff
@@ -0,0 +1,111 @@
+# rotatekey Service
+
+Rotates the crypt4gh encryption key of a file that is mapped to a dataset and stored in the SDA.
+
+## Service Description
+
+The rotatekey service re-encrypts the header of a file with the configured target key, and updates the database with the new header and encryption key hash.
+
+When running, rotatekey reads messages from the `rotatekey_stream` RabbitMQ queue.
+For each message, these steps are taken (errors halts progress, the message is Nack'ed, an info-error message is sent and the service moves on to the next message):
```

Comment:
Not very clear to me. Do the steps mean those included in the parenthesis or listed in the bullet points below?